packages/travix-ui-kit/tests/unit/stepper/stepper.spec.js

```diff
@@ -0,0 +1,160 @@
+import { shallow } from 'enzyme';
+import React from 'react';
+import Stepper from '../../../components/stepper/stepper';
+
+const event = {
+  preventDefault: jest.fn(),
+};
+
+describe('Stepper', () => {
+  describe('#render()', () => {
+    it('should render component', () => {
+      const wrapper = shallow(
+        <Stepper
+          initValue={2}
+          minValue={1}
+        />
+      );
+
+      expect(wrapper).toMatchSnapshot();
+    });
+
+    it('should render with disabled decrement button', () => {
+      const wrapper = shallow(
+        <Stepper
+          initValue={1}
+          minValue={1}
+        />
+      );
+
+      expect(wrapper).toMatchSnapshot();
+      expect(wrapper.find('.ui-stepper__button-decrease').hasClass('ui-stepper__button-disabled')).toBe(true);
+    });
+
+    it('should render with disabled increment button', () => {
+      const wrapper = shallow(
+        <Stepper
+          initValue={3}
+          maxValue={3}
+        />
+      );
+
+      expect(wrapper).toMatchSnapshot();
+      expect(wrapper.find('.ui-stepper__button-increase').hasClass('ui-stepper__button-disabled')).toBe(true);
+    });
+
+    it('should reset current value if initValue prop changes (2 -> 4)', () => {
+      const wrapper = shallow(
+        <Stepper
+          initValue={2}
+        />
+      );
+
+      expect(wrapper).toMatchSnapshot();
+      expect(wrapper.find('.ui-stepper__value').text()).toEqual('2');
```

Comment:
Do you need to match the snapshot _and_ check the contents of a specific node? Seems redundant to me.